### DISCUSSION TITLE
P1 Radiation Model Questions

### DISCUSSION THREAD

**Discussion by ChinmayU78** [su2code/SU2#1309] [Q&A]
Hi,
I am trying to use the P1 radiation model capability in SU2 for a Mach 6 air flow, however I looked at the original pull request when the P1 model was implemented in SU2 and a comment mentions that the model is not valid for compressible flow. I was able to run a Mach 6 case with the radiative heat transfer model turned on, so I was wondering if it has been updated to work with compressible flows or if it is still only valid for incompressible flows? In addition, I was wondering what absorption and scattering coefficients would be appropriate for air using this model? Can both coefficients be set to 0 if air is assumed to be transparent?
Thanks,
Chinmay

![image](https://user-images.githubusercontent.com/74565109/123012210-7dd3a480-d387-11eb-8f87-8ebfd90cc92b.png)

**Comment by bigfooted** [comment]
Looks like the radiation source term is not added in CEulerSolver, only in CInEulerSolver. It might be sufficient to add it in CEulerSolver.
Regarding absorption coefficient, this is temperature and wavelength dependent and also depends on what you consider to be air. Only nitrogen and oxygen at room temperature? I would take 0 then. If it contains some water or CO2, then it's different. And for scattering, there are some simple analytic functions that you could use as an estimate, but it is much different when non-pure air is considered.

**Reply by ChinmayU78** [reply]
Thanks for the clarification, I will have to determine the best way to proceed. I am currently using the RANS solver, so would it be fairly simple to modify to add to that? I don't have much experience with modifying the SU2 code. 
I will most likely consider the air to be transparent as well. 
Thanks for the help!
